ote/src/clj/ote/services/service_search.sql

```diff
@@ -34,20 +34,9 @@ SELECT COUNT(id) FROM "transport-service" WHERE published IS NOT NULL;
 
 -- name: total-company-count
 -- single?: true
--- Count the total number of companies providing transport services
--- this includes companies reported in CSV files
-SELECT COUNT(*)
-  FROM (SELECT DISTINCT op."business-id"
-          FROM "transport-service" ts
-          JOIN "transport-operator" op ON ts."transport-operator-id" = op.id
-         WHERE ts.published IS NOT NULL
-           AND op."deleted?" = FALSE
-        UNION
-        SELECT DISTINCT (x.c)."business-id"
-          FROM (SELECT unnest(COALESCE(sc.companies, ts.companies)) c
-                  FROM "transport-service" ts
-                  LEFT JOIN service_company sc ON sc."transport-service-id" = ts.id
-                 WHERE ts.published IS NOT NULL) x) y;
+-- Count the total number of companies providing transport services or participating in providing
+SELECT COUNT(x."business-id")
```

Comment:
👍 

Just mentioning that this seems to be faster (~75ms) to execute than the often referred alternative
```
    SELECT COUNT (DISTINCT "business-id")
    FROM "all-companies";
```
which executes around 88-100ms